static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -63,6 +66,91 @@ function BreadcrumbItem({
 
   const forceSpan = 'category' in frame && FRAMES_WITH_BUTTONS.includes(frame.category);
 
+  const renderDescription = useCallback(() => {
+    return typeof description === 'string' ||
+      (description !== undefined && isValidElement(description)) ? (
+      <Description title={description} showOnlyOnOverflow isHoverable>
+        {description}
+      </Description>
+    ) : (
+      <InspectorWrapper>
+        <ObjectInspector
+          data={description}
+          expandPaths={expandPaths}
+          onExpand={onInspectorExpanded}
+          theme={{
+            TREENODE_FONT_SIZE: '0.7rem',
+            ARROW_FONT_SIZE: '0.5rem',
+          }}
+        />
+      </InspectorWrapper>
+    );
+  }, [description, expandPaths, onInspectorExpanded]);
+
+  const renderComparisonButton = useCallback(() => {
+    return frame?.data && 'mutations' in frame.data ? (
+      <div>
+        <OpenReplayComparisonButton
+          replay={replay}
```

Comment:
FYI one of us will have a minor conflict here

This is the other PR https://github.com/getsentry/sentry/pull/73095/files#diff-82c19ad3589826b66a3468b8cc00fffb9202b6b5936f7910c20e9480be9b21ed